sky/packages/sky/lib/src/rendering/box.dart

```diff
@@ -31,12 +46,19 @@ class BoxConstraints extends Constraints {
     this.maxHeight: double.INFINITY
   });
 
+  final double minWidth;
+  final double maxWidth;
+  final double minHeight;
+  final double maxHeight;
+
+  /// Constructs a box constraints that is respected only by size
```

Comment:
`by the given size` or find some way to make `size` typographically denote the argument, because otherwise that's confusing (it could mean, e.g., respected only by size but not by position?)